app/models/user.rb

```diff
@@ -15,4 +16,21 @@ def self.digest string
            end
     BCrypt::Password.create string, cost: cost
   end
+
+  def self.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
```suggestion
    update_column :remember_digest, User.digest(remember_token)
```

ri hì

giải thích cho chị nghe tại sao chị suggest là thế này

Reply:
`update_column` không validate, không gọi các `callback`, không gọi method `save`, nên khi dùng `update_column` tránh được callback throw error